flink-core/src/test/java/org/apache/flink/testutils/DeeplyEqualsChecker.java

```diff
@@ -90,8 +88,6 @@ private boolean deepEquals0(Object e1, Object e2) {
 			return deepEqualsArray(e1, e2);
 		} else if (e1 instanceof Tuple && e2 instanceof Tuple) {
 			return deepEqualsTuple((Tuple) e1, (Tuple) e2);
-		} else if (e1 instanceof Row && e2 instanceof Row) {
```

Comment:
This is not needed anymore because the Row itself now does the deep equals, right?

Reply:
Yes, the Row implementation supports that now. And if a custom implementation is still needed the checker allows to override it if necessary.